charts/goosefs/docker/init-users/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM centos:centos8.2.2004
```

Comment:
Do we really need runtime-specific scripts to do initialization like `init-users.sh` or `chmod_fuse_mountpoint.sh`?

Reply:
In terms of maintainability and extensibility, I feel like  runtime-specific will be better. WDYT @cheyang